common/laws/00_health_system.txt

```diff
@@ -39,11 +39,16 @@ law_charitable_health_system = {
 	unlocking_technologies = {
 		medical_degrees
 	}	
+	modifier = {
+		country_institution_health_system_max_investment_add = 2
```

Comment:
So as discussed ideally this would be a thing that increases conditioned on religious policies. But I don't think conditional synergies is a thing we should consider this update round.